mindsdb_native/libs/controllers/transaction.py

```diff
@@ -282,6 +317,48 @@ def run(self):
 
                     output_data[f'{predicted_col}_confidence'][row_number] = probability_true_prediction
 
+            # confidence estimation
+            self.lmd['all_conformal_ranges'] = {}
+            for predicted_col in self.lmd['predict_columns']:
+                X = deepcopy(self.input_data.data_frame)
+                for col in self.lmd['columns_to_ignore'] + self.lmd['predict_columns']:
+                    X.pop(col)
+
+                if not self.lmd['stats_v2']['is_classification'][predicted_col]:
```

Comment:
I don't think it's worth making key `is_classification` becuase `stats_v2[col]['typing']['data_type'] == DATA_TYPES.CATEGORICAL` is simple enough. It works either way though.